qadence_protocols/mitigations/protocols.py

```diff
@@ -34,6 +35,8 @@ def __call__(
             module = importlib.import_module(PROTOCOL_TO_MODULE[self.protocol])
         except KeyError:
             ImportError(f"The module for the protocol {self.protocol} is not implemented.")
+        except (ModuleNotFoundError, ImportError) as e:
```

Comment:
I guess you can group them all. Besides the `type(e)` re-issues the caught exception. Up to you if you'd like to do it. Also, I think the `from e` is superfluous.